src/workerd/api/util.c++

```diff
@@ -69,10 +70,10 @@ namespace {
 
 template <typename Func>
 auto translateTeeErrors(Func&& f) -> decltype(kj::fwd<Func>(f)()) {
-  try {
+  KJ_TRY {
```

Comment:
An example of `KJ_TRY` and `KJ_CATCH` at work.

Ultimately, I think I prefer the raw syntax, myself.